spec/models/rss_feed_url_spec.rb

```diff
@@ -115,6 +115,20 @@
         end
       end
     end
+
+    context 'when the last_crawl_status is greater than 255 characters' do
+      before do
+        rss_feed_url.update(last_crawl_status: 'x' * 500)
```

Comment:
I would change this so that you are not making any database calls, which always slow down tests. Instead of updating a database record, you could change the context to:
let(:rss_feed_url) { RssFeedUrl.new(last_crawl_status: 'x' * 500) }

Then your test can be changed to:
`expect{ rss_feed_url.valid? }.to change{ rss_feed_url.length }.from(500).to(255)`

I like to use `to change` matchers because it ensures that your test setup is correct.